Library/Homebrew/livecheck/strategy/electron_builder.rb

```diff
@@ -0,0 +1,75 @@
+# typed: true
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      # The {ElectronBuilder} strategy fetches content at a URL and parses
+      # it as an electron-builder appcast in YAML format.
+      #
+      # @api private
+      class ElectronBuilder
+        extend T::Sig
+
+        NICE_NAME = "electron-builder"
+
+        # A priority of zero causes livecheck to skip the strategy. We do this
+        # for {ElectronBuilder} so we can selectively apply the strategy using
+        # `strategy :electron_builder` in a `livecheck` block.
+        PRIORITY = 0
+
+        # The `Regexp` used to determine if the strategy applies to the URL.
+        URL_MATCH_REGEX = %r{^https?://.+?/.+?\.(yml|atom)}i.freeze
```

Comment:
> The second commit modifies the `URL_MATCH_REGEX` to allow URLs pointing to `.atom` files, which seem to be supported by [`find-appcast`](https://github.com/Homebrew/homebrew-cask/blob/master/developer/bin/find-appcast). I'll remove this if it isn't supposed to be the case here.

Someone more familiar with homebrew/cask may have a better idea but, from what I'm seeing, `electron-builder` appcasts may only use a `latest-mac.yml` file. If that's true, we should restrict `URL_MATCH_REGEX` accordingly. Something like this:

```suggestion
        URL_MATCH_REGEX = %r{^https?://.+/latest-mac.yml$}i.freeze
```

Reply:
I have only ever seen `.atom` for GitHub releases.